editor/editor_inspector.h

```diff
@@ -46,6 +46,8 @@ class PopupMenu;
 class SpinBox;
 class TextureRect;
 
+class StyleBoxFlat;
```

Comment:
This should probably be added to the list with others, don't see much reason to put it separately.

Reply:
Possible this happened during the rebase.

Reply:
S goes before T though :)

Reply:
Indeed :)